lms/templates/learner_dashboard/program_details_tab_view.underscore

```diff
@@ -48,9 +49,24 @@
                     </div>
                 <% } %>
                 <% if (isSubscriptionEligible) { %>
-                <div class="d-flex flex-column align-items-start flex-sm-row  align-items-sm-center upgrade-subscription">
-                    <a href="<%- subscriptionUrl %>" class="btn-brand btn cta-primary upgrade-button">
-                    <% if (subscriptionState === 'inactive') { %>
+                <div class="d-flex flex-column align-items-start flex-xl-row  align-items-xl-center upgrade-subscription">
+                    <a
+                        href="<%- subscriptionUrl %>"
+                        class="js-subscription-cta btn-brand btn cta-primary upgrade-button"
+                        <% if (subscriptionState === 'active') { %>
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        <% } %>
```

Comment:
Is there a reason why the button would _only_ have these attributes when the subscriptionState is 'active'?

Reply:
Ah, actually, I _think_ I understand. Given the difference in the icon being used for 'active' and 'inactive', we only intend to open a new tab to redirect the user if they need to `Manage my subscription` and redirect within the same tab if they're simply `Restart my subscription`. But _both_ will lead the user to the same subscription URL. Is this intention simply then due to the difference in icon and the accessibility meaning behind `launchIcon`? Is it possible that we intend for the button to open to a new tab no matter what?

Secondly, I notice the icon is to the _right_ when subscription is 'active', and it is to the _left_ when 'inactive'. From a design perspective, do we want to maintain consistency by keeping icons to the right, and isn't that the norm for buttons anyway?

Reply:
When the subs are active, button says manage my subscription
since this opens a URL of stripe customer portal we want this to open in a new tab,
And we add these attributes to make that happen,
additionally there's an icon in the button to indicate this link opens in a new tab

Reply:
Gotcha! The makes sense knowing now where the user is redirected to. 
Did you have any thoughts about the second point on placement of the icon for 'active' and 'inactive'?

Reply:
> But both will lead the user to the same subscription URL
Actually that assumption is incorrect since [the subscription URL changes base on the subscription state](https://github.com/openedx/edx-platform/blob/1088dc224ff50c3ba4d69ca04438491b52c6b300/lms/static/js/learner_dashboard/models/program_subscription_model.js#L28-L32)

Reply:
The norm is indeed to have icons for buttons on the left, but a common exception to that case is icon for external links